### ISSUE TITLE
Not showing up inside certain iron-pages

### ISSUE THREAD

**Issue by JaySunSyn** [sorin-davidoi/fullcalendar-calendar#3]
If the page of `fullcalendar-calendar` is not the first page of iron-pages when the page loads initially, the calendar doesn't show up. To make it show up, you have to change e.g. the view.

```
<iron-pages selected="0">
  <section>Page 1</section>
  <section><fullcalendar-calendar></fullcalendar-calendar></section>
  <section>Page 3</section>
</iron-pages>
```

![bildschirmfoto 2016-06-20 um 12 29 01](https://cloud.githubusercontent.com/assets/5872432/16183294/c2968b10-36e2-11e6-9bcd-da636aa1613f.png)

Thanks for your work @sorin-davidoi!

**Comment by sorin-davidoi** [comment]
Fixed in [0.2.3](https://github.com/sorin-davidoi/fullcalendar-calendar/releases/tag/v0.2.3).

**Comment by JaySunSyn** [comment]
great!
